p2panda-store/src/lib.rs

```diff
@@ -22,6 +22,8 @@
 //! by default.
 #[cfg(feature = "memory")]
 pub mod memory_store;
```

Comment:
I'd suggest renaming this from `memory_store` to `memory`.

Then we have `p2panda_store::{memory, sqlite};`

Or we rather go with `memory_store` and `sqlite_store`.

What do you think?

Reply:
I like this suggesting :+1: